README.md

```diff
@@ -5,7 +5,7 @@ This project aims to create simple songbook for storing, managing and viewing so
 
 First clone the repository from Github and switch to the new directory:
 
-    $ git clone git@github.com/pehala/song-book.git
+    $ git clone https://github.com/pehala/song-book
```

Comment:
If you want to change it to HTTPS instead of SSH then this is the correct link `https://github.com/pehala/song-book.git`.